android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.java

```diff
@@ -173,6 +178,11 @@ public void onUpdate() {
     mToolbar.setNavigationOnClickListener(mBackClickListener);
 
 
+    // we apply padding if status bar is translucent
```

Comment:
You could use same constant as in `RequiresApi` annotation in `applyWindowInsets` method